shell/platform/windows/flutter_window.cc

```diff
@@ -0,0 +1,286 @@
+#include "flutter/shell/platform/windows/flutter_window.h"
+#include <chrono>
+
+namespace flutter {
+
+#define WINVER 0x0605
+
+FlutterWindow::FlutterWindow() {
+  surface_manager = std::make_unique<AngleSurfaceManager>();
+}
+
+FlutterWindow::FlutterWindow(const char* title,
+                             const int x,
+                             const int y,
+                             const int width,
+                             const int height) noexcept
+    : FlutterWindow() {
+  Win32Window::Initialize(title, x, y, width, height);
+}
+
+FlutterWindow::~FlutterWindow() {
+  DestroyRenderSurface();
+}
+
+FlutterDesktopWindowControllerRef FlutterWindow::SetState(FlutterEngine eng) {
+  engine_ = eng;
+
+  // TODO: Restructure the internals to follow the structure of the C++ API,
```

Comment:
Does this apply to this implementation? I don't see a forest of interrelated structs and classes here like the GLFW version has evolved.

Reply:
I realized later that the structs still exist, and are just in the other file. Since this part is all temp code anyway as written, how about just removing the TODO here since that's not the right longer term change for this copy of the code.

Reply:
Done in next commit